storage/migrations/0009_oasis_3_emerald_init.up.sql

```diff
@@ -0,0 +1,90 @@
+-- Indexer state initialization for the Emerald ParaTime, after the Damask Upgrade.
+
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS oasis_3.emerald_rounds
+(
+  height    NUMERIC PRIMARY KEY,
+  version   BIGINT,
+  timestamp NUMERIC NOT NULL,
+
+  block_hash      TEXT NOT NULL,
+  prev_block_hash TEXT NOT NULL,
+
+  io_root          TEXT NOT NULL,
+  state_root       TEXT NOT NULL,
+  messages_hash    TEXT NOT NULL,
+  in_messages_hash TEXT NOT NULL,
+
+  -- Arbitrary additional data.
+  extra_data JSON
+);
+
+CREATE TABLE IF NOT EXISTS oasis_3.emerald_transactions
```

Comment:
should there be more data here?

Reply:
I'll just remove the table. Nothing writes to it anyway, until the follow-up to this PR, which will add emerald transaction processing semantics beyond modules, is created.

Removing feels cleaner than trying to patch the migration later.